dace/frontend/python/newast.py

```diff
@@ -3970,7 +3970,7 @@ def _parse_sdfg_call(self, funcname: str, func: Union[SDFG, SDFGConvertible], no
         # Change transient names
         arrays_before = list(sdfg.arrays.items())
         for arrname, array in arrays_before:
-            if array.transient and arrname[:5] == '__tmp':
+            if array.transient and arrname[:5] == '__tmp' and arrname not in sdfg.size_arrays():
```

Comment:
Is there any case where a size array can have a `__tmp` name instead of `__dace_deferred`?